sonar-project.properties

```diff
@@ -2,8 +2,10 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-sonar.projectKey=alliander-opensource_power-grid-model-io
-sonar.organization=alliander-opensource
+# sonar.projectKey=alliander-opensource_power-grid-model-io
```

Comment:
Hm, apparently this is an issue that we should solve... I don't know much about sonar... do you?
Or do you have an idea, @TonyXiang8787?

Reply:
Sonar cloud should be skipped for external pull requests, so it should not be necessary to supply your own credentials.